Documentation/RELEASE_NOTES.md

```diff
@@ -17,6 +17,7 @@
 Enhance several license file descriptions to include the file name
 Updated SDWAN portal to support 20.11 version
 Force rebuild after missing files (METROAE-298) 
+VSD deploy no longer idempotent (METROAE-304)
```

Comment:
How about "Fixed VSD deploy task to be idempotent"?

Reply:
done